docs/pre-commit.md

```diff
@@ -31,10 +31,10 @@ pre-commit hooks, append the following configuration to
 the `.pre-commit-config.yaml` file:
 
 ```yaml
-  - repo: https://github.com/jackdewinter/pymarkdown
```

Comment:
interesting...  I did use the word append, but having it in the sample would be useful.  how about:
```
repos:
  - repo: https://github.com/jackdewinter/pymarkdown
    rev: main
    hooks:
      - id: pymarkdown
```

Reply:
I didn't remove the word "append". Do you think I should?